2022-02-01 17:38:57,955 DEBUG server.py Была вызвана функция create_arg_parser c параметрами (), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-02-01 17:38:57,955 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-02-01 17:39:08,591 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 3373)
2022-02-01 17:39:08,591 DEBUG server.py Получено сообщение {'action': 'presence', 'time': 'Tue Feb  1 17:39:08 2022', 'user': {'account_name': 'Guest'}}
2022-02-01 17:39:08,591 DEBUG server.py Была вызвана функция process_client_message c параметрами ({'action': 'presence', 'time': 'Tue Feb  1 17:39:08 2022', 'user': {'account_name': 'Guest'}},), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-02-01 17:39:08,591 INFO server.py Cформирован ответ клиенту {'response': 200}
2022-02-01 17:39:08,591 DEBUG server.py Соединение с клиентом ('127.0.0.1', 3373) закрывается.
2022-02-01 17:40:37,134 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 3426)
2022-02-01 17:40:37,134 DEBUG server.py Получено сообщение {'action': 'presence', 'time': 'Tue Feb  1 17:40:37 2022', 'user': {'account_name': 'Guest'}}
2022-02-01 17:40:37,134 DEBUG server.py Была вызвана функция process_client_message c параметрами ({'action': 'presence', 'time': 'Tue Feb  1 17:40:37 2022', 'user': {'account_name': 'Guest'}},), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-02-01 17:40:37,134 INFO server.py Cформирован ответ клиенту {'response': 200}
2022-02-01 17:40:37,134 DEBUG server.py Соединение с клиентом ('127.0.0.1', 3426) закрывается.
2022-02-01 17:43:30,117 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 3528)
2022-02-01 17:43:30,117 ERROR server.py Не удалось декодировать JSON строку, полученную от клиента ('127.0.0.1', 3528). Соединение закрывается.
2022-02-01 17:44:16,752 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 3567)
2022-02-01 17:44:16,752 ERROR server.py Не удалось декодировать JSON строку, полученную от клиента ('127.0.0.1', 3567). Соединение закрывается.
2022-02-01 17:44:30,534 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 3591)
2022-02-01 17:44:30,550 DEBUG server.py Получено сообщение {'action': 'presence', 'time': 'Tue Feb  1 17:44:30 2022', 'user': {'account_name': 'Guest'}}
2022-02-01 17:44:30,550 DEBUG server.py Была вызвана функция process_client_message c параметрами ({'action': 'presence', 'time': 'Tue Feb  1 17:44:30 2022', 'user': {'account_name': 'Guest'}},), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-02-01 17:44:30,550 INFO server.py Cформирован ответ клиенту {'response': 200}
2022-02-01 17:44:30,550 DEBUG server.py Соединение с клиентом ('127.0.0.1', 3591) закрывается.
2022-02-01 17:48:47,687 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 3768)
2022-02-01 17:48:47,687 DEBUG server.py Получено сообщение {'action': 'presence', 'time': 'Tue Feb  1 17:48:47 2022', 'user': {'account_name': 'Guest'}}
2022-02-01 17:48:47,687 DEBUG server.py Была вызвана функция process_client_message c параметрами ({'action': 'presence', 'time': 'Tue Feb  1 17:48:47 2022', 'user': {'account_name': 'Guest'}},), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-02-01 17:48:47,687 INFO server.py Cформирован ответ клиенту {'response': 200}
2022-02-01 17:48:47,687 DEBUG server.py Соединение с клиентом ('127.0.0.1', 3768) закрывается.
2022-02-01 17:56:47,280 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 4027)
2022-02-01 17:56:47,280 ERROR server.py Не удалось декодировать JSON строку, полученную от клиента ('127.0.0.1', 4027). Соединение закрывается.
2022-02-01 18:00:00,736 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 4140)
2022-02-01 18:00:00,736 ERROR server.py Не удалось декодировать JSON строку, полученную от клиента ('127.0.0.1', 4140). Соединение закрывается.
2022-02-01 18:03:48,447 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 4323)
2022-02-01 18:03:48,447 DEBUG server.py Получено сообщение {'action': 'presence', 'time': 'Tue Feb  1 18:03:48 2022', 'user': {'account_name': 'Guest'}}
2022-02-01 18:03:48,447 DEBUG server.py Была вызвана функция process_client_message c параметрами ({'action': 'presence', 'time': 'Tue Feb  1 18:03:48 2022', 'user': {'account_name': 'Guest'}},), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-02-01 18:03:48,447 INFO server.py Cформирован ответ клиенту {'response': 200}
2022-02-01 18:03:48,447 DEBUG server.py Соединение с клиентом ('127.0.0.1', 4323) закрывается.
2022-02-01 18:05:29,914 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 4383)
2022-02-01 18:05:29,914 DEBUG server.py Получено сообщение {'action': 'presence', 'time': 'Tue Feb  1 18:05:29 2022', 'user': {'account_name': 'Guest'}}
2022-02-01 18:05:29,914 DEBUG server.py Была вызвана функция process_client_message c параметрами ({'action': 'presence', 'time': 'Tue Feb  1 18:05:29 2022', 'user': {'account_name': 'Guest'}},), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-02-01 18:05:29,914 INFO server.py Cформирован ответ клиенту {'response': 200}
2022-02-01 18:05:29,914 DEBUG server.py Соединение с клиентом ('127.0.0.1', 4383) закрывается.
2022-02-01 18:06:00,682 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 4413)
2022-02-01 18:06:00,682 DEBUG server.py Получено сообщение {'action': 'presence', 'time': 'Tue Feb  1 18:06:00 2022', 'user': {'account_name': 'Guest'}}
2022-02-01 18:06:00,682 DEBUG server.py Была вызвана функция process_client_message c параметрами ({'action': 'presence', 'time': 'Tue Feb  1 18:06:00 2022', 'user': {'account_name': 'Guest'}},), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-02-01 18:06:00,682 INFO server.py Cформирован ответ клиенту {'response': 200}
2022-02-01 18:06:00,682 DEBUG server.py Соединение с клиентом ('127.0.0.1', 4413) закрывается.
2022-02-01 18:07:56,633 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 4477)
2022-02-01 18:07:56,633 DEBUG server.py Получено сообщение {'action': 'presence', 'time': 'Tue Feb  1 18:07:56 2022', 'user': {'account_name': 'Guest'}}
2022-02-01 18:07:56,633 DEBUG server.py Была вызвана функция process_client_message c параметрами ({'action': 'presence', 'time': 'Tue Feb  1 18:07:56 2022', 'user': {'account_name': 'Guest'}},), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-02-01 18:07:56,633 INFO server.py Cформирован ответ клиенту {'response': 200}
2022-02-01 18:07:56,633 DEBUG server.py Соединение с клиентом ('127.0.0.1', 4477) закрывается.
2022-02-01 18:09:20,508 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 4533)
2022-02-01 18:09:20,508 DEBUG server.py Получено сообщение {'action': 'presence', 'time': 'Tue Feb  1 18:09:20 2022', 'user': {'account_name': 'Guest'}}
2022-02-01 18:09:20,508 DEBUG server.py Была вызвана функция process_client_message c параметрами ({'action': 'presence', 'time': 'Tue Feb  1 18:09:20 2022', 'user': {'account_name': 'Guest'}},), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-02-01 18:09:20,508 INFO server.py Cформирован ответ клиенту {'response': 200}
2022-02-01 18:09:20,508 DEBUG server.py Соединение с клиентом ('127.0.0.1', 4533) закрывается.
2022-02-01 18:11:06,424 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 4594)
2022-02-01 18:11:06,424 DEBUG server.py Получено сообщение {'action': 'presence', 'time': 'Tue Feb  1 18:11:06 2022', 'user': {'account_name': 'Guest'}}
2022-02-01 18:11:06,424 DEBUG server.py Была вызвана функция process_client_message c параметрами ({'action': 'presence', 'time': 'Tue Feb  1 18:11:06 2022', 'user': {'account_name': 'Guest'}},), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-02-01 18:11:06,424 INFO server.py Cформирован ответ клиенту {'response': 200}
2022-02-01 18:11:06,424 DEBUG server.py Соединение с клиентом ('127.0.0.1', 4594) закрывается.
2022-02-01 18:11:25,956 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 4620)
2022-02-01 18:11:25,956 DEBUG server.py Получено сообщение {'action': 'presence', 'time': 'Tue Feb  1 18:11:25 2022', 'user': {'account_name': 'Guest'}}
2022-02-01 18:11:25,956 DEBUG server.py Была вызвана функция process_client_message c параметрами ({'action': 'presence', 'time': 'Tue Feb  1 18:11:25 2022', 'user': {'account_name': 'Guest'}},), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-02-01 18:11:25,956 INFO server.py Cформирован ответ клиенту {'response': 200}
2022-02-01 18:11:25,956 DEBUG server.py Соединение с клиентом ('127.0.0.1', 4620) закрывается.
2022-02-01 18:11:43,597 DEBUG server.py Была вызвана функция create_arg_parser c параметрами (), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-02-01 18:11:43,597 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-02-01 18:12:01,776 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 4661)
2022-02-01 18:12:01,776 DEBUG server.py Получено сообщение {'action': 'presence', 'time': 'Tue Feb  1 18:12:01 2022', 'user': {'account_name': 'Guest'}}
2022-02-01 18:12:01,776 DEBUG server.py Была вызвана функция process_client_message c параметрами ({'action': 'presence', 'time': 'Tue Feb  1 18:12:01 2022', 'user': {'account_name': 'Guest'}},), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-02-01 18:12:01,776 DEBUG server.py Разбор сообщения от клиента -----------------: ({'action': 'presence', 'time': 'Tue Feb  1 18:12:01 2022', 'user': {'account_name': 'Guest'}},)
2022-02-01 18:12:01,776 INFO server.py Cформирован ответ клиенту {'response': 200}
2022-02-01 18:12:01,776 DEBUG server.py Соединение с клиентом ('127.0.0.1', 4661) закрывается.
